client/app/scripts/superdesk-highlights/highlights.js

```diff
@@ -165,77 +165,49 @@
     function HighlightsTitleDirective(highlightsService, $timeout) {
         return {
             scope: {
-                item: '=item',
-                orientation: '=?'
+                item: '=item'
             },
             templateUrl: 'scripts/superdesk-highlights/views/highlights_title_directive.html',
             // todo(petr): refactor to use popover-template once angular-bootstrap 0.13 is out
             link: function(scope, elem) {
-                var unmarkBox = elem.find('.unmark').hide(),
-                    icon = elem.find('i'),
-                    isOpen,
-                    closeTimeout;
 
-                scope.orientation = scope.orientation || 'right';
+                /*
+                 * Set 'open' class on dropdown menu element
+                 * @param {string} isOpen
```

Comment:
Ah, of course, I meant just the opposite, i.e. the method _sets_ a value but does _not_ toggle it. Thanks! :)